src/box/lua/tuple.h

```diff
@@ -131,6 +131,13 @@ luaT_pushtupleornil(struct lua_State *L, struct tuple *tuple)
 	return 1;
 }
 
+/**
+ * Retrieves tuple format from Lua stack, and throws an error if object at
+ * specified index is not a tuple format.
+ */
+struct tuple_format *
+lbox_check_tuple_format(struct lua_State *L, int idx);
```

Comment:
AFAIU the convention for `lbox_` functions is to conform to the [lua_CFunction](https://www.lua.org/manual/5.1/manual.html#lua_CFunction) type, i.e. accept one `struct lua_State *` parameter and return `int`.

All other our Lua related function are named `luaT_*`.

It is not strictly followed in our code, AFAIR, but seems as a good rule to follow in the new code.

Reply:
Applied your suggestion.